src/extension.ts

```diff
@@ -25,7 +25,7 @@ export function activate(context: vscode.ExtensionContext) {
         treeActions.openNoteFromId(id);
     });
 
-    vscode.commands.registerCommand('code-annotation.summary', () => {
+    vscode.commands.registerCommand('code-annotation.summary', async () => {
```

Comment:
Is there any reason for this?

Reply:
I suspect I had a reason for it when adding / removing the commands, but that shouldn't have changed, I must have missed it somehow when committing, fixing now.

Reply:
It looks great! I'll just run some manual tests, but should be merged in a few hours.